docs/osf-trolling-list/jsm_documentation.md

```diff
@@ -0,0 +1,92 @@
+# Jira Service Management client procedure
+
+## Dasharo Support Package
+
+If you are interested in starting cooperation using the Support Package, it is the perfect time
+to start. The Support Package will help us allocate resources and a secure timeline for your
+needs. To ensure that we can provide comprehensive assistance with your current products and
+future new developments, we encourage you to choose our new Support Package for more streamlined
+cooperation, guaranteed availability, and quick, on-time delivery.
+
+## How to add a request
+
+### Raising requests using the portal
+
+To add a new request using the web service:
+
+1. Go to [portal](https://jira-3mdeb.atlassian.net/servicedesk/customer/portals) and select the
+   project.
+2. Select a type of request and provide a required data. You can also add attachements by drag
+   and dropping into any part of the web page, or by clicking the `Browse` button.
+3. Click send. Your request is created and awaits our response.
+
+![](../images/jsm_add_request.png)
+
+![](../images/jsm_view_requests.png)
+
+### Raising requests using email
+
+If you prefer raising a request using email, you can do this by sending a message to special
+email address generated for the project.
+
+The email address is be provided to you by separate message.
+
+### Commenting a request and adding files
+
+You can always add a comment to the request and attach files. You can do it directly from the
+JSM request page or using email simply by replying to the email thread.
+
+You'll also recieve an information, w
+
+## Process and request statuses
+
+![](../images/jsm_workflow.png)
+
+### Open
+
+Your request is created and visible for us, but we didn't start working on it yet.
+
+### Analysis
+
+Our team is analyzing your request. We will soon let you know what will be the cost of resolving
+this request.
+
+### Waiting for client
+
+Your action is required to continue working on your request.
```

Comment:
I suggest adding information on what situation we will be waiting for client response - e.g. after our comment with SP estimation we will ask for acceptance, to start the implementation or after we ask for a clarification from Client.

Reply:
I have added this information